packages/cxl-ui/scss/cxl-theme.scss

```diff
@@ -0,0 +1,61 @@
+@import "./utils/mq";
+
+html,
+body {
+  height: 100%;
+  min-height: 100%;
+}
+
+a {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.sub-title {
+  margin-top: var(--lumo-space-tall-l);
+  text-align: center;
+}
+
+.flex {
+  display: flex;
+}
+
+strong {
+  color: var(--lumo-primary-text-color);
+}
+
+.align {
+  &-left {
+    text-align: left;
+  }
+  &-center {
+    text-align: center;
+  }
+  &-right {
+    text-align: right;
+  }
+}
+
+.screen-reader-text {
+  display: none;
+}
+
+.wrap {
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: calc(var(--lumo-space-m) / 2);
+  padding-right: calc(var(--lumo-space-m) / 2);
+
+  @include mq($from: desktop) {
+    max-width: 960px;
+    width: 960px;
```

Comment:
No hard-coded `width` values, this makes `.wrap` only usable in specific outer container cases, and that's not the intention.